test/builder/QuarkBuilderCometWithdraw.t.sol

```diff
@@ -496,4 +496,19 @@ contract QuarkBuilderCometWithdrawTest is Test, QuarkBuilderTest {
             paymentUsdc_(maxCosts) // user will pay for transaction with withdrawn funds, but it is not enough
         );
     }
+
+    function testCometWithdrawCostTooHigh() public {
+        PaymentInfo.PaymentMaxCost[] memory maxCosts = new PaymentInfo.PaymentMaxCost[](2);
+        maxCosts[0] = PaymentInfo.PaymentMaxCost({chainId: 1, amount: 5e6});
+        maxCosts[1] = PaymentInfo.PaymentMaxCost({chainId: 8453, amount: 5e6});
+        QuarkBuilder builder = new QuarkBuilder();
+
+        vm.expectRevert(abi.encodeWithSelector(Actions.NotEnoughFundsToBridge.selector, "usdc", 4e6, 4e6));
```

Comment:
Shouldn't this be "USDC"? (capitalized)

Reply:
That's good question... haha. 
I agree.. it should be capitalized,  let me double check... looks like all existing tests is verifying it with lower cased 😮 ...

Reply:
Weird that the check is case insensitive 🤔

Reply:
oh wow... it's case sensitive...  hmm I think I need to dig a bit deeper to see why some tests seems still carrying side effect of `toLowerCase()` and some don't.